File: program_data/github_java_sort_function/10/103.java
package com.levon.algorithms.sorting;

public class BucketSort {
	
	public static void main(String[] args) {
		int[] ar = new int[] {44,7,2,3,1,9,6,4,8};
		for (int i = 0; i < ar.length; i++) {
			System.out.print(ar[i]+" ");
		}
		System.out.println();
		int max = findMaxElement(ar);
		int[] bucket = new int[max+1];
		for (int i = 0; i < bucket.length; i++) {
			bucket[i] = 0;
		}
		for (int i = 0; i < ar.length; i++) {
			bucket[ar[i]]++;
		}
		int k = 0;
		for (int j = 0; j < bucket.length; j++) {
			while(bucket[j] > 0) {
				ar[k] = j;
				bucket[j]--;
				k++;
			}	
		}
		for (int i = 0; i < ar.length; i++) {
			System.out.print(ar[i]+" ");
		}
	}

	private static int findMaxElement(int[] ar) {
		int max = ar[0];
		for (int i = 1; i < ar.length; i++) {
			if(ar[i] > max) {
				max = ar[i];
			}
		}
		return max;
	}
}

File: program_data/PKU_raw/82/103.c
int main()
{
	int i,n,p1[100],p2[100],t[100],c=0,max=0;
	scanf("%d",&n);
	for(i=0;i<n;i++){
	     scanf("%d%d",&p1[i],&p2[i]);
		 if(((p1[i]>=90)&&(p1[i]<=140))&&((p2[i]>=60)&&(p2[i]<=90)))
			 t[i]=1;
		 else
			 t[i]=0;
	}
	for(i=0;i<n;i++){
		if(t[i]==1){
		     c++;
			 if(c>max)
				 max=c;
		}
		else{
		    c=0;
		}
	}
	printf("%d",max);
	return 0;
}